otelcolbuilder/.otelcol-builder.yaml

```diff
@@ -197,23 +197,23 @@ replaces:
   #         <autogenerated>:1 +0xcb fp=0xc0005c5058 sp=0xc0005c4fb8 pc=0x44c1fab
   # go.opentelemetry.io/collector/service/internal/builder.attachReceiverToPipelines({0x9acc778, 0xc00012a000}, {{0xc0000f5d40, {0x9a92fc0, 0xc000b250a0}, {0x9a9e020, 0xca100b8}, 0x0}, {{0x8fe90d1, 0xc}, ...}}, ...)
-  #         go.opentelemetry.io/collector@v0.47.0/service/internal/builder/receivers_builder.go:191 +0xc83 fp=0xc0005c5238 sp=0xc0005c5058 pc=0x7bd2583
+  #         go.opentelemetry.io/collector@v0.46.0/service/internal/builder/receivers_builder.go:191 +0xc83 fp=0xc0005c5238 sp=0xc0005c5058 pc=0x7bd2583
   # go.opentelemetry.io/collector/service/internal/builder.(*receiversBuilder).buildReceiver(0xc0005c54f8, {0x9acc778, 0xc00012a000}, {{0xc0000f5d40, {0x9a92fc0, 0xc000b250a0}, {0x9a9e020, 0xca100b8}, 0x0}, {{0x8fe90d1, ...}, ...}}, ...)
-  #         go.opentelemetry.io/collector@v0.47.0/service/internal/builder/receivers_builder.go:257 +0x325 fp=0xc0005c53f8 sp=0xc0005c5238 pc=0x7bd29a5
+  #         go.opentelemetry.io/collector@v0.46.0/service/internal/builder/receivers_builder.go:257 +0x325 fp=0xc0005c53f8 sp=0xc0005c5238 pc=0x7bd29a5
   # go.opentelemetry.io/collector/service/internal/builder.BuildReceivers({0xc0000f5680, {0x9a92fc0, 0xc000b250a0}, {0x9a9e020, 0xca100b8}, 0x0}, {{0x8fe90d1, 0xc}, {0x91094d4, 0x2f}, ...}, ...)
-  #         go.opentelemetry.io/collector@v0.47.0/service/internal/builder/receivers_builder.go:109 +0x437 fp=0xc0005c5680 sp=0xc0005c53f8 pc=0x7bd1257
+  #         go.opentelemetry.io/collector@v0.46.0/service/internal/builder/receivers_builder.go:109 +0x437 fp=0xc0005c5680 sp=0xc0005c53f8 pc=0x7bd1257
   # go.opentelemetry.io/collector/service.newService(0xc000fe3980)
-  #         go.opentelemetry.io/collector@v0.47.0/service/service.go:74 +0x5ce fp=0xc0005c58a0 sp=0xc0005c5680 pc=0x7be8dae
+  #         go.opentelemetry.io/collector@v0.46.0/service/service.go:74 +0x5ce fp=0xc0005c58a0 sp=0xc0005c5680 pc=0x7be8dae
   # go.opentelemetry.io/collector/service.(*Collector).setupConfigurationComponents(0xc00016e460, {0x9acc778, 0xc00012a000})
-  #         go.opentelemetry.io/collector@v0.47.0/service/collector.go:200 +0x31b fp=0xc0005c5a28 sp=0xc0005c58a0 pc=0x7be679b
+  #         go.opentelemetry.io/collector@v0.46.0/service/collector.go:200 +0x31b fp=0xc0005c5a28 sp=0xc0005c58a0 pc=0x7be679b
   # go.opentelemetry.io/collector/service.(*Collector).Run(0xc00016e460, {0x9acc778, 0xc00012a000})
-  #         go.opentelemetry.io/collector@v0.47.0/service/collector.go:240 +0x24e fp=0xc0005c5b48 sp=0xc0005c5a28 pc=0x7be6aae
+  #         go.opentelemetry.io/collector@v0.46.0/service/collector.go:240 +0x24e fp=0xc0005c5b48 sp=0xc0005c5a28 pc=0x7be6aae
   # go.opentelemetry.io/collector/service.NewCommand.func1(0xc000a52280, {0x8fd4b1d?, 0x2?, 0x2?})
```

Comment:
We shouldn't change that

Reply:
This was the original version I believe. In any case: I've tested this and we don't need this anymore after bump to 0.48.0.

Reply:
Exactly, that's why I changed it to the original. I assume this shouldn't have been from 0.46.0 to 0.47.0 in the first place.